package.json

```diff
@@ -15,7 +15,7 @@
     },
     "preview": true,
     "categories": [
-        "Languages",
+        "Programming Languages",
```

Comment:
why need change this?

Reply:
VSCode has changed the category name. But I think they will finally work out a solution for the compatibility issue.

Reply:
The extension doesn't support any language service feature, such as `intellisense` and `snippet`. We could remove this tag, and add it back if we add such features in future.